README.md

```diff
@@ -102,19 +102,20 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Animals table]**
+- **[Schema]**
+- **[Queries]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-
+<!-- 
 > Add a link to your deployed project.
```

Comment:
- [ ] [OPTIONAL] I suggest you remove this section all together because you're not using a live demo

Reply:
Thanks for direction.